### PR TITLE
ACE3-Action: Stereo, inside Vehicles

### LINKED ISSUE #1328: ACE³ self interactions for interaction with radios not available if sitting in vehicle
**Arma 3 Version:** `current` (stable)
**CBA Version:** `3.4.1.170912`
**Task Force Radio Version:** `1.0.264`
**ACE3 Version:** `3.11.0`

**Mods:**
- `@CBA_A3`
- `@task_force_radio`
- `@ace`

**Description:**
- The self interactions "Radios" and "Lower Headset"/"Raise Headset" aren't shown when sitting in vehicle.

**Steps to reproduce:**
- Get in vehicle
- Try to lower your headset with ace³ self interaction.

### PR DESCRIPTION
**Pull request by Dorbedo**
* the stereo settings added to the self interaction menu
* fix for the default value in TFAR_fnc_getSwStereo
* add the possibility to interact from inside vehicles, on chairs
* fixes #1328

### CODE REVIEW

**Comment by dedmen** [comment]
The stereo action only shows the 2 stereo actions that are not currently active right?
So if I am on LEFT then it will only show Center and Right?

**Comment by Dorbedo** [comment]
Exactly.

**Comment by dedmen** [comment]
If we add more actions for the local radio the stereo settings should be in a submenu. So that
Equipment -> Radios -> PRC148->Stereo setting->Left

File: addons/core/ACEActions.hpp [hunk 1]
```diff
@@ -5,23 +5,23 @@ class CAManBase: Man {
             class TFAR_Radio {
                 displayName = "Radios";
                 condition = "(([] call TFAR_fnc_haveSWRadio)||([] call TFAR_fnc_haveLRRadio))";
-                exceptions[] = {"isNotInside","isNotSitting"};
+                exceptions[] = {"isNotInside", "isNotSitting", "isNotSwimming"};
```
Comment (dedmen) [inline addons/core/ACEActions.hpp]: We want that while swimming tho.. so "isNotSwimming" ? Or am I confusing stuff?

Comment (Dorbedo) [reply addons/core/ACEActions.hpp]: This way the `isNotSwimming` condition is not evaluated. This way, you can be swimming while interacting.
Ref:
https://github.com/acemod/ACE3/blob/master/addons/medical/ACE_Medical_SelfActions.hpp#L12

**Comment by Dorbedo** [comment]
Can be merged from my side.

**Pull request merged**